support-frontend/assets/helpers/storage/abandonedBasketCookies.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import * as cookie from 'helpers/storage/cookie';
+
+export function useAbandonedBasketCookie(
+	product: string, 
+	amount: number, 
+	billingPeriod: string, 
+	region: string,
+    cookieExpiryDays: number
```

Comment:
should this be a param? I feel like it could just be a const in this file if it's 3 in both places

Reply:
Agreed, thanks